src/components/CommitteeDescriptions/Descriptions/description.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+export default class Description extends React.Component {
+	render() {
+		return (
+			<div className="committee-component">
+				<div className="content-section left">
+					<div className="ornament image-ornament">
+						<img src={this.props.description.image} />
+					</div>
+				</div>
+				<div className="content-section right">
+					<div className="text-section">
```

Comment:
same here, try to remove any unnecessary divs, spans, etc. Try to make the code clean and minimal p much